waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -42,6 +52,10 @@ const Logo = styled.img.attrs({
   width: 260px;
   height: 150px;
   margin-bottom: -15px; // required to make up for slight gap in the svg
```

Comment:
After this line, can you add `margin-right` so that there's more space between the logo and "Contact Us" section? From what I tested, `margin-right: 20px` seems sufficient, but feel free to try different values. This can also be kept outside of `@media`.